src/components/play/fillInTheBlanks/Blank.tsx

```diff
@@ -21,18 +20,13 @@ export const WordBox = styled(Typography)<WordBoxProps>(
     showCorrectness,
     isCorrect,
     isReadonly,
-    hasChanged,
     filled,
   }) => {
     let bgColor = backgroundColor ?? 'transparent';
-    if (!hasChanged) {
-      if (showCorrection || showCorrectness) {
-        bgColor = isCorrect
-          ? theme.palette.success.main
-          : theme.palette.error.main;
-      } else {
-        bgColor = theme.palette.warning.main;
-      }
+    if (showCorrection || showCorrectness) {
+      bgColor = isCorrect
+        ? theme.palette.success.main
```

Comment:
Maybe use `light` instead of `main` to improve the contrast between text and background ? Maybe in another PR ?

Reply:
I can do that now :)